src/schematools/types.py

```diff
@@ -594,7 +624,7 @@ def model_name(self) -> str:
         model_name = self.id
         if self.dataset.version is not None and not self.dataset.is_default_version:
             model_name = f"{model_name}_{self.dataset.version}"
-        return to_snake_case(model_name)
+        return str(to_snake_case(model_name))  # mypy pleaser
```

Comment:
Should not be necessary. `to_snake_case` is already type annotated to return a `str`.

Reply:
Yes:
    returning Any from function declared to return "str"  [no-any-return]
Now solved it with a `cast`. I think you are right (other comment) that the return type of `to_snake_case`, because of the decorator, cannot be inferred correctly.